From 72552b70b8c8a54091cbde91f0ffc8b3f310d52b Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 28 Apr 2014 12:34:02 +0800
Subject: [PATCH 311/602] ENGR00242365 Update BUILD_ID for KK4.4.2_2.0.0-Alpha
 release

Update BUILD_ID as "2.0.0-alpha-rc1"
Update BUILD_NUMBER as "20140520" to align new release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 sabresd_6sx/build_id.mk    | 4 ++--
 8 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 013f595..8ee7cd7 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 013f595..8ee7cd7 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index 013f595..8ee7cd7 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index 013f595..8ee7cd7 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 013f595..8ee7cd7 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 013f595..8ee7cd7 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 013f595..8ee7cd7 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
diff --git a/sabresd_6sx/build_id.mk b/sabresd_6sx/build_id.mk
index 013f595..8ee7cd7 100644
--- a/sabresd_6sx/build_id.mk
+++ b/sabresd_6sx/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-alpha-rc2
-export BUILD_NUMBER=20140110
+export BUILD_ID=2.0.0-alpha-rc1
+export BUILD_NUMBER=20140520
-- 
1.8.0

